.travis.yml

```diff
@@ -9,35 +11,30 @@ python:
 cache: ccache
 compiler: gcc
 
-before_install: # Use this to prepare the system to install prerequisites or dependencies
```

Comment:
I don't think, you can remove these install instructions. moveit_ci is executed in docker and thus, doesn't install anything to the Travis image. These instructions here are required to setup ROS in Travis.

Reply:
this PR attempts to use moveit_ci which already has these things in the docker image

Reply:
You didn't get my point: moveit_ci is only used for building the examples, but not for building the docs. For the latter, Travis is used directly, without any docker! Hence, you need to install these dependencies.